test/snippet/CMakeLists.txt

```
@@ -0,0 +1,74 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+cmake_minimum_required (VERSION 3.10)
+project (sharg_test_snippet CXX)
+
+find_package (SHARG REQUIRED HINTS ${CMAKE_SOURCE_DIR}/../../build_system)
+
+include (${CMAKE_SOURCE_DIR}/../../lib/seqan3/test/seqan3-test.cmake)
+include (${CMAKE_SOURCE_DIR}/../../lib/seqan3/test/cmake/diagnostics/list_unused_snippets.cmake)
+
+add_library (snippet_main snippet_main.cpp)
+target_link_libraries (snippet_main PUBLIC seqan3::test gtest)
+
+macro (sharg_snippet test_name_prefix snippet snippet_base_path)
+    seqan3_test_component (snippet_target_name "${snippet}" TARGET_NAME)
+    seqan3_test_component (snippet_test_name "${snippet}" TEST_NAME)
+    seqan3_test_component (snippet_target_path "${snippet}" TARGET_PATH)
+    set (target "${snippet_target_name}_snippet")
+
+    add_executable (${target} "${snippet_base_path}/${snippet}")
+    target_link_libraries (${target} PUBLIC snippet_main sharg::sharg)
+    set_target_properties(${target}
+        PROPERTIES
+        RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${snippet_target_path}"
+    )
+    collect_used_snippet (${target})
+
+    set (snippet_test_target "${test_name_prefix}/${snippet_test_name}_snippet")
+    add_test (NAME "${snippet_test_target}" COMMAND ${target})
+
+    # disable version checker, as it interferes with comparing the snippet output
+    set_tests_properties ("${snippet_test_target}" PROPERTIES ENVIRONMENT SEQAN3_NO_VERSION_CHECK=0)
```

```suggestion
    set_tests_properties ("${snippet_test_target}" PROPERTIES ENVIRONMENT SEQAN3_NO_VERSION_CHECK=1)
```

Shouldn't this be `1`??

---

I copied this from seqan3 :thinking: 1 means "do not check" right? So yeah I think it should be 1. Why negate the varieble :see_no_evil: